src/game.js

```diff
@@ -209,6 +218,14 @@
             // reset the frame counter
             frameCounter = 0;
             frameRate = ~~(0.5 + 60 / me.sys.fps);
+            
+            // set step size based on the updatesPerSecond
+            stepSize = (1000 / Math.min(me.sys.updatesPerSecond, me.sys.fps));
```

Comment:
Assume one update takes 2ms and one draw takes 30ms.

`me.timer.getDelta()` will return at least 32.

You have made the assumption above that the game is running at a perfect 60fps in the line:
`accumulator += me.timer.getDelta();` accumulator =~ 16.6667